src/store/store.js

```diff
@@ -0,0 +1,10 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from 'store/reducers/rootReducer';
+
+const composeEnhancers =
+    (typeof window !== 'undefined' &&
+        window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) ||
+    compose;
+
+export const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunk)));
```

Comment:
hm, from what I see, you will have devtools enabled not only for `dev` build but for `prod` as well. Do we really need it there? You can use `process.env.NODE_ENV` here to exclude it from `prod` build

Reply:
correct me if I'm wrong 🤔